src/main/java/com/redhat/devtools/lsp4ij/launching/UserDefinedLanguageServerSettings.java

```diff
@@ -119,6 +119,8 @@ public static class UserDefinedLanguageServerItemSettings {
 
         private String initializationOptionsContent;
 
+        private String clientConfigurationContent = "{}";

```

Comment:
The default value of `{}` here is what allows smooth backward-compatibility with existing language servers.